client/src/pages/CardInfo-components/CardInfo.js

```diff
@@ -0,0 +1,184 @@
+  const [showDeadline, setShowDeadline] = useState(true);
+  const [showComment, setShowComment] = useState(true);
+
+  // description section handlers
+  const addDescription = () => {
+    setShowDescription(true);
+  };
+  const handleSaveDescription = (description) => {
+    setInfo({ ...info, description });
+  };
+  const handleDeleteDescription = () => {
+    setShowDescription(false);
+  };
+
+  // deadline section handlers
+  const addDeadline = () => {
+    setShowDeadline(true);
+  };
+  const handleSaveDeadline = (deadline) => {
+    setInfo({ ...info, deadline });
+  };
+  const handleDeleteDeadline = () => {
+    setShowDeadline(false);
+  };
+
+  // comment section handlers
+  const addComment = () => {
+    setShowComment(true);
+  };
+  const handleSaveComment = (comment) => {
+    setInfo({ ...info, comment });
+  };
+  const handleDeleteComment = () => {
+    setShowComment(false);
+  };
+
+  return (
+    <Container
+      maxWidth={false}
+      className={`${classes.bg} ${
+        showCardInfo ? classes.dBlock : classes.dNone
+      }`}
+    >
+      <Card className={classes.cardContainer}>
+        <CardContent className={classes.header}>
+          <Box className={classes.titleContainer}>
+            <Assignment color="primary" style={{ marginRight: "3px" }} />
+            <Typography className={`${classes.marginRight} ${classes.title}`}>
+              {task.title}
+            </Typography>
+            <Box
+              className={`${classes.cardStatus} ${classes.marginRight}`}
+            ></Box>
+          </Box>
+          <Typography
+            variant="body1"
+            style={{ color: "gray" }}
+            className={classes.marginLeft}
+          >
+            In List "Math"
+          </Typography>
+          <Button className={classes.closeCard} onClick={() => closeCardInfo()}>
+            &times;
+          </Button>
+        </CardContent>
+        <CardContent className={classes.cardContent}>
+          <Box m={0} className={`${classes.cardBody} ${classes.left}`}>
+            <CardInfoDescription
+              saveDescription={handleSaveDescription}
+              showDescription={showDescription}
+              deleteDescription={handleDeleteDescription}
+            />
+            <CardInfoDeadline
+              saveDeadline={handleSaveDeadline}
+              showDeadline={showDeadline}
+              deleteDeadline={handleDeleteDeadline}
+            />
+            <CardInfoComment
+              saveComment={handleSaveComment}
+              showComment={showComment}
+              deleteComment={handleDeleteComment}
+            />
+          </Box>
+          <Box m={0} className={`${classes.cardBody} ${classes.right}`}>
+            <Box mt={1.5}>
+              <Typography variant="subtitle2" align="center">
+                ADD TO CARD:
+              </Typography>
+              <Button
+                variant="contained"
```

Comment:
Try removing this, I think it would make the UI cleaner to have them not be contained